registration/verify.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Libera Chat account verification
+absolutify: true
+---
+
+{% raw %}
+{::nomarkdown}
+<!-- markdownlint-disable MD033 -->
+
+<div class="verification">
+  <form method="POST" action="{{target}}">
+    <input type="submit" value="Verify">
+  </form>
+  <script>
+    /*
```

Comment:
You probably want to uncomment this js before merging?

Reply:
yes, you're right, but also is that js going to work any more? it feels like i lost the ID it's referring to

Reply:
Ah, yeah. You need to add that back